### DISCUSSION TITLE
build fail

### DISCUSSION THREAD

**Discussion by wsrer** [flet-dev/flet#3984] [Q&A]
### Duplicate Check

- [X] I have searched the [opened issues](https://github.com/flet-dev/flet/issues) and there are no duplicates

### Describe the bug

![image](https://github.com/user-attachments/assets/e498fea4-ff43-417a-9025-f28f377ad29b)

### Code sample

<details open><summary>Code</summary>

```python
import flet
from flet import IconButton, Page, Row, TextField, icons


def main(page: Page):
    page.title = "Flet counter example"
    page.vertical_alignment = "center"

    txt_number = TextField(value="0", text_align="right", width=100)

    def minus_click(e):
        txt_number.value = str(int(txt_number.value) - 1)
        page.update()

    def plus_click(e):
        txt_number.value = str(int(txt_number.value) + 1)
        page.update()

    page.add(
        Row(
            [
                IconButton(icons.REMOVE, on_click=minus_click),
                txt_number,
                IconButton(icons.ADD, on_click=plus_click),
            ],
            alignment="center",
        )
    )


flet.app(target=main)

```

</details>

### To reproduce

just run
```shell
flet build windows
```

### Expected behavior

build success

### Screenshots / Videos

<details open>
<summary>Captures</summary>

![image](https://github.com/user-attachments/assets/e498fea4-ff43-417a-9025-f28f377ad29b)

</details>

### Operating System

Windows

### Operating system details

win11

### Flet version

0.22.1

### Regression

Yes, it used to work in a previous Flet version (please specify the version in additional details)

### Logs

<details open><summary>Logs</summary>

```console
 Islatan@reika E:\....\flet-demo1  flet build windows --verbose
Flutter bootstrap directory: C:\Users\Islatan\AppData\Local\Temp\flet_flutter_build_z469PrEbc2
Additional Flutter dependencies:
{}
Creating Flutter bootstrap project from gh:flet-dev/flet-build-template with ref 0.22.1...OK
Customizing app icons and splash images...Copying E:\workspace\py\flet-demo1\assets\icon.png to C:\Users\Islatan\AppData\Local\Temp\flet_flutter_build_z469PrEbc2\images
OK
Generating app icons...
Run subprocess: ['E:\\flutter\\bin\\dart.BAT', 'run', 'flutter_launcher_icons']
Error: Unable to find git in your PATH. 
Deleting Flutter bootstrap directory C:\Users\Islatan\AppData\Local\Temp\flet_flutter_build_z469PrEbc2

Run subprocess: ['E:\\flutter\\bin\\flutter.BAT', '--version']
Error building Flet app - see the log of failed command above.

```

</details>

**Comment by ndonkoHenri** [comment]
Search how to solve this online: https://stackoverflow.com/questions/51263438/how-to-solve-unable-to-find-git-in-your-path-on-flutter
